### ISSUE TITLE
Add "The Circuit" Defib Data Tooling

### ISSUE THREAD

**Issue by andrewbolster** [andrewbolster/bolster#1124]
https://www.bhf.org.uk/defibdata

**Comment by andrewbolster** [comment]
## Data Source Evaluation: The Circuit — BHF National Defibrillator Network
- **Source**: https://www.bhf.org.uk/how-you-can-help/how-to-save-a-life/defibrillators/national-defibrillator-network-the-circuit/defibrillator-location-data
- **Format**: Excel (.xlsx) bulk download, refreshed on the first working day of each month; no API/CSV/REST endpoint
- **PxStat matrix**: not applicable — this is a UK-wide British Heart Foundation dataset, not a NISRA publication, so it is out of scope for the PxStat API (checked `https://data.nisra.gov.uk/`, no equivalent matrix exists)
- **Accessibility**: 2/5 — attempted to fetch both `bhf.org.uk/defibdata` and the direct download-data page; both returned HTTP 403 (bot/Cloudflare-style protection), so the actual download URL and column schema could not be confirmed programmatically in this session. Even where accessible, this is a single monthly XLSX snapshot (no incremental/API access), and the BHF terms explicitly require **express prior written consent from BHF** before building any app or service on top of the data — this is a hard legal gate independent of technical access.
- **Recommendation**: MAYBE
- **Next steps**: For data-build — do not proceed until BHF written consent is confirmed (contact via the site's data-request process, e.g. ndnsupport@bhf.org.uk). Once consent is obtained: download the XLSX manually to document the real column schema (location/coordinates, availability/access type, guardian contact, last-checked date), filter to NI (BT postcodes) to match the library's regional focus, and treat the "may be inaccurate/incomplete" disclaimer as a driver for `validate_data()` checks (e.g. postcode format, non-null coordinates). Since there's no API, `get_latest_data()` would need to re-download the full monthly file each run (`_base.download_file()` with caching, not `pxstat.read_dataset()`). This is a new domain (BHF, not NISRA/PSNI/DVA) so it would likely warrant a new top-level `data_sources/bhf/` package rather than fitting an existing one.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by andrewbolster** [comment]
Staleness check — this is the oldest open issue in the tracker (2024-04-03), by a wide margin (next-oldest open issue is from 2026-04-29).

Re-reading the evaluation: the real blocker isn't technical (the XLSX download and NI/BT-postcode filtering are both straightforward) — it's that BHF's data terms require **written consent before building any tool or service** on top of "The Circuit" data, and there's no indication that consent was ever requested.

Still wanted? If so, the next concrete step is emailing BHF for consent, not more technical scoping — everything technical here is already answered. If not (or if consent was never realistically going to happen), worth closing rather than letting it keep aging in the backlog.
